app/move/fields/create.js

```diff
@@ -11,6 +11,16 @@ const assessmentQuestionComments = {
   },
 }
 
+const requiredAssessmentQuestionComments = {
+  ...assessmentQuestionComments,
+  label: {
+    text: 'fields::assessment_comment.required',
+    classes: 'govuk-label--s',
+  },
+  dependent: null, // This value is populated within the assessment controller
```

Comment:
I was thinking about this earlier and it being a bit of weird concept and I wonder if actually what we could do is use the presence of `validate: 'required'` to set the dependent object in the map function?

The dependance only occurs when this field is pulled in as a child within the assessment controller so maybe checking for that assessment question being required feels like a more logical check than checking for `null`. What do you think?